src/contacts/ViewContact/util.js

```diff
@@ -21,3 +20,15 @@ export const deleteContact = (orgMutator, contactMutator, contactId, org) => {
 
   return Promise.resolve();
 };
+
+export const saveContact = (contactMutator, contact) => {
+  let method = contactMutator.POST;
+
+  if (contact.id) {
+    method = contactMutator.PUT;
+  }
+
+  method(contact);
+
+  return Promise.resolve();
```

Comment:
could you please describe how it suppose to work?

Reply:
Fixed